main.go

```diff
@@ -3,11 +3,89 @@ package main
 import (
 	"github.com/gin-gonic/gin"
 	"net/http"
+	"log"
+	//"database/sql"
+	
+	_ "github.com/lib/pq"
+		"github.com/jmoiron/sqlx"
 )
 
-func main() {
-	router := gin.Default()
-	
+var router = gin.Default()
+var db *sqlx.DB
+
+
+/*
+ * Initialise the database as a global variable
+ */
+func initDatabase() {
+	url := "postgres://ghuvahbe:8Y7zUWnG1VZSF8vRsZPvo_W-AdD1EVkP@arjuna.db.elephantsql.com:5432/ghuvahbe"
+	var err error //Seperate initialisation allows use of = instead of := in next line
+	//Thus we modify the global variable db instead of declaring a local one with :=
+	db, err = sqlx.Open("postgres", url)
+	if err!= nil {
+		log.Fatalln(err)
+	}
+	db.Ping()
+}
+
+/*
+ * Return a json of all patients
+ */
+func GetAllPatients(c *gin.Context){
+	type Patient struct {
+		Id int `db:"id"`
```

Comment:
Also specify the `json` naming for each of [these attributes](https://github.com/riskycase/my-go-app/pull/2/files#diff-7ddfb3e035b42cd70649cc33393fe32cR36-R40).
```suggestion
		Id int `json="id" db:"id"`
```